pkg/apis/autoscaling/register.go

```diff
@@ -83,6 +83,19 @@ const (
 	// scale-to-zero-pod-retention-period global setting.
 	ScaleToZeroPodRetentionPeriodKey = GroupName + "/scaleToZeroPodRetentionPeriod"
 
+	// MetricAggregationAlgorithm is the annotation that can be used for selection
+	// of the algorithm to use for averaging metric data in the Autoscaler.
+	// Since autoscalers are a pluggable concept, this field is only validated
+	// for Revisions that are owned by Knative Pod Autoscaler.
+	// Possible values for KPA are:
+	// - empty/missing or "linear" — linear average over the whole
+	//   metric window (default);
+	// - weightedExponential — weighted average with exponential decay.
+	//   KPA will compute the decay multiplier automatically and it is capped
```

Comment:
I'd like to expand "automatically" somehow, is there a good way to explain the criteria it uses to calculate the multiplier in half a sentence? :). "Automatically based on the size of the window" or something?

Reply:
done.